python/lsst/afw/geom/ellipses/_baseCore.cc

```diff
@@ -0,0 +1,73 @@
+/*
+ * LSST Data Management System
+ * Copyright 2008-2016  AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/eigen.h"
+#include "pybind11/stl.h"
+#include <lsst/utils/python.h>
+
+#include "ndarray/pybind11.h"
+
+#include "lsst/afw/geom/ellipses/BaseCore.h"
+#include "lsst/afw/geom/ellipses/Convolution.h"
+#include "lsst/afw/geom/ellipses/Transformer.h"
+
+namespace py = pybind11;
+namespace lsst {
+namespace afw {
+namespace geom {
+namespace ellipses {
+void wrapBaseCore(lsst::utils::python::WrapperCollection &wrappers) {
+    wrappers.wrapType(py::class_<BaseCore, std::shared_ptr<BaseCore>>(wrappers.module, "BaseCore"),
+                      [](auto &mod, auto &cls) {
+                          cls.def("__eq__", &BaseCore::operator==, py::is_operator());
+                          cls.def("__nq__", &BaseCore::operator!=, py::is_operator());
+
+                          /* Members */
+                          cls.def("getName", &BaseCore::getName);
+                          cls.def("clone", &BaseCore::clone);
+                          cls.def("normalize", &BaseCore::normalize);
+                          cls.def("grow", &BaseCore::grow);
+                          cls.def("scale", &BaseCore::scale);
+                          cls.def("getArea", &BaseCore::getArea);
+                          cls.def("getDeterminantRadius", &BaseCore::getDeterminantRadius);
+                          cls.def("getTraceRadius", &BaseCore::getTraceRadius);
+                          //    clsBaseCore.def("transform", (typename BaseCore::Transformer const
+                          //    (BaseCore::*)(lsst::geom::LinearTransform const &) const)
+                          //    &BaseCore::transform); clsBaseCore.def("getGridTransform",
+                          //    &BaseCore::getGridTransform);
```

Comment:
I'm sure this is preexisting even if it doesn't look like it, but would you mind just deleting these commented-out lines, and those below (and the similar ones in a few of the other ellipses files)?